packages/ui/src/components/IconButton/IconButton.test.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { ChatIcon } from '@chakra-ui/icons';
+import { render, screen } from '@testing-library/react';
+
+import { IconButton } from './IconButton';
+
+describe('IconButton', () => {
+  it('renders correctly', () => {
+    render(<IconButton data-testid="icon-btn" icon={<ChatIcon />} />);
+
+    const iconButton = screen.getByTestId('icon-btn');
```

Comment:
You should use test id only as a last resort. For icon buttons catching them with aria-label would be a better idea. As a bonus, you will test the component's accessibility.